CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -804,9 +810,198 @@ private static void OnTimeoutEvent(object source, ElapsedEventArgs e)
+                    else if (playerBets.Min() != _currentBet) //condition if more than one player can not match the current bet
+                    {
+                        //all-in bet added to the pot, other players matched bets added to pot as well
+                        allInBet = playerBets.Min();
+                        AddToPot(allInBet, Enumerable.Repeat(allInBet, playerBets.Count() - bets).ToList(), playersActive);
+
+                        //subtracting all-in bet value from all other bets, the all-in bet value has already been added to a pot
+                        for (int i = 0; i < playerBets.Count(); i++)
+                        {
+                            playerBets[i] -= allInBet; // Subtract the amount put into pot from each bet
+                            if (playerBets[i] < 0) //prevents any players with lesser all-in bets from having negetive value bets for future calculations in this function
+                            {
+                                playerBets[i] = 0;
+                            }
+                        }
+                        //create a list of players that are ineligible for the new main pot
+                        List<int> allInPlayers = playerBets.Select((value, index) => new { value, index })
+                                   .Where(x => x.value == 0)
+                                   .Select(x => x.index)
+                                   .ToList();
+                        playersActive.RemoveAll(value => allInPlayers.Contains(value));
+                        playerBets.RemoveAll(value => value == 0);
+
+                        //subtract the amount added to the side pot from the current bet
+                        _currentBet -= allInBet;
+
+                        //New side pot created. Pot that all-in player can win is shifted to inactive position, no more bets can be added to this pot
+                        Pots.Add(new Pot(PotType.SIDE, Pots[0].Total, Pots[0].EligiblePlayers));
+                        Pots[0].EligiblePlayers = playersActive;
+                        Pots[0].Total = 0; //reseting active pot value to empty
+                    }
+                    else
+                    {
+                        //add remainder of bets from players not all-in into new main pot
+                        AddToPot(_currentBet, playerBets, playersActive);
+                    }
+                }
+            }
+            /// <summary>
+            /// Pays out the winnings to the player for the individual pot. 
+            /// </summary>
+            /// <param name="winners">Number of players that have won the pot.</param>
+            public int DistributePot(int winners)
+            {
+                int _payout = Pots.Last().Total / winners; //splits payout if more than 1 winner is present
+                Pots.RemoveAt(Pots.Count - 1);
+                return _payout;
+            }
+            public List<int> GetPotAmounts()
```

Comment:
summary comment. this is an attribute, so it should be at the top of the class, not within the methods.